src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Models/Properties/Resources.resx

```diff
@@ -414,7 +414,7 @@ Please contact Microsoft for further assistant.</value>
     <value>Please provide AzureRmBackupManagementServer of BackupEngineType as DpmEngine or DpmVenusEngine and provide BackupManagementType as SCDPM. Provided AzureRmBackupManagementServer has BackupEngineType {0} and backupManagementType {1} which is not valid.</value>
   </data>
   <data name="UnsupportedContainerException" xml:space="preserve">
-    <value>Please provide Container of containerType as Windows and backupManagementType as MARS. Provided Container has containerType {0} and backupManagementType {1} which is invalid.</value>
+    <value>Please provide Container of containerType as Windows and backupManagementType as MARS or Container of containerType as AzureSQL and backupManagementType as AzureSQL. Provided Container has containerType {0} and backupManagementType {1} which is invalid.</value>
```

Comment:
Please consider something like:
'Please provide either 1. a Container object whose container type is "Windows" and backup management type is "MARS" or 2. a Container object whose container type is "AzureSQL" and backup management type is "AzureSQL". Provided Container object's container type is "{0}" and backup management type is "{1}" which is invalid.'